### PR TITLE
Phishing Campaign And Compliance

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
- [x] In Progress - Testing.

## Related Issues
Related to: https://github.com/demisto/etc/issues/33810
fixes: https://github.com/demisto/etc/issues/38473
fixes: https://github.com/demisto/etc/issues/33847
fixes: https://github.com/demisto/etc/issues/36082
fixes: https://github.com/demisto/etc/issues/36989

##### Phishing Incident
- Added a section linking to related phishing campaigns.
- Reorganized the layout.

##### Phishing Investigation - Generic v2
- Added a playbook for detection and management of phishing campaigns.
- Added an email notification for the user if they were the target of a phishing campaign attack.
- Malicious email addresses will now be blocked in the Block Indicators - Generic v2 subplaybook.
- Fixed a bug where file detonation results were not consistently enriched.
- Consolidated the enrichment subplaybooks. The Entity Enrichment - Phishing v2 playbook is now used instead.
- Added an option to search and delete emails using the O365 - Security And Compliance - Content Search playbook

## Minimum version of Cortex XSOAR
6.0.0

## Does it break backward compatibility?
No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by idovandijk** [comment]
@richardbluestone can you please review:

Pack readme - line 20 (link to tutorial):
https://github.com/demisto/content/blob/0654a3b7c8f06b4bcfaec06eeb759547a5513838/Packs/Phishing/README.md

Playbook - playbook inputs:
https://github.com/demisto/content/blob/0654a3b7c8f06b4bcfaec06eeb759547a5513838/Packs/Phishing/Playbooks/Phishing_Investigation_-_Generic_v2_-_6_0.yml

**Review by michalgold — APPROVED** [review]

**Comment by idovandijk** [comment]
@bakatzir @yaakovi can one of you review the dynamic section script please? I understand we need a TL to review the code, we need to merge this ASAP

File: Packs/Phishing/Scripts/LinkToPhishingCampaign/LinkToPhishingCampaign.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+BLACK_COLOR = "color:rgb(64,65,66)"
+
+server_url = ""
+html = ""
+
+
+try:
+    incident_id = demisto.incidents()[0].get('id', {})
+    incident = demisto.incidents()
```
Comment (JasBeilin) [inline Packs/Phishing/Scripts/LinkToPhishingCampaign/LinkToPhishingCampaign.py:12]:
```suggestion
    incident = demisto.incidents()
    incident_id = incident[0].get('id', {})
```

File: Packs/Phishing/Playbooks/Phishing_Investigation_-_Generic_v2_README.md [hunk 2]
```diff
@@ -24,62 +26,63 @@ These task have the following names:
...
-* Email Address Enrichment - Generic v2.1
+* Entity Enrichment - Phishing v2
+* Search And Delete Emails - Generic v2
+* Block Indicators - Generic v2
+* Detect & Manage Phishing Campaigns
 
-#### Integrations
-* Builtin
+### Integrations
+This playbook does not use any integrations.
```
Comment (bakatzir) [inline Packs/Phishing/Playbooks/Phishing_Investigation_-_Generic_v2_README.md:53]: are you sure? it's sub playbooks do.
I'm not saying that we need all the list though.

Comment (idovandijk) [reply Packs/Phishing/Playbooks/Phishing_Investigation_-_Generic_v2_README.md:53]: The SDK apparently does not drill-down, anymore and doesn't write `* Builtin`.
I will add it back

**Review by bakatzir — APPROVED** [review]

**Pull request merged**